### ISSUE TITLE
[FEATURE] Sign application

### ISSUE THREAD

**Issue by Cheesebaron** [robvdpol/RaceControl#394]
**Is your feature request related to a problem? Please describe.**
Installer/Application is not signed causing Windows to show a nasty dialog warning users that the Application might be malicious.

**Describe the solution you'd like**
Obtain a code signing certificate to sign the Application.

**Comment by robvdpol** [comment]
Possibly not needed when moving to Squirrel.

**Comment by Cheesebaron** [comment]
Using a different installer won't change the fact that the App is not signed. You will still see the Windows warning dialog even using Squirrel.

So in any case, signing the App would be a much better experience for users.

**Comment by robvdpol** [comment]
I'm looking into puchasing a code signing certificate, Certum seems to be the most affordable: https://shop.certum.eu/open-source-code-signing.html

They have a specific (cheap) option for open source software, but I'm not sure whether this will immediately get rid of the Windows SmartScreen warning.

**Comment by jdschuitemaker** [comment]
Since you are are developer, you might already know this, still this StackOverflow article might give some insight about the Windows Smartscreen warning:

(https://stackoverflow.com/questions/48946680/how-to-avoid-the-windows-defender-smartscreen-prevented-an-unrecognized-app-fro)

**Comment by Cheesebaron** [comment]
> I'm looking into puchasing a code signing certificate, Certum seems to be the most affordable: https://shop.certum.eu/open-source-code-signing.html
> 
> They have a specific (cheap) option for open source software, but I'm not sure whether this will immediately get rid of the Windows SmartScreen warning.

Yeah, not on the first installation, but smart screen will be trained by the users who still accept and install the Application and eventually it will not complain anymore.
